Entity/AddressEmbeddable.php

```diff
@@ -103,64 +103,64 @@ public function setLocale(string $locale): void
         $this->locale = $locale;
     }
 
-    public function setAdditionalName(string $additionalName): void
+    public function setAdditionalName(?string $additionalName): void
```

Comment:
I don't see the reason for accepting nulls if we need strings. It will work, but solving client problems should be solved on the client side. If the client side doubts whether the variable is a string or null, the simplest solution is to cast it to a string. But it should be done on the other side.

Reply:
As we are not using custom data transformer, default one form symfony is used and if the value is empty string, it sets `null`:  
https://github.com/symfony/symfony/blob/7.1/src/Symfony/Component/Form/Form.php#L998

For example, with a french address, i have this kind of error without this fix:  
> Uncaught PHP Exception Symfony\Component\PropertyAccess\Exception\InvalidArgumentException: "Expected argument of type "string", "null" given at property path "organization"." at vendor/symfony/property-access/PropertyAccessor.php line 274

I tried to use `preserve_empty_strings` option but it did not work.

Reply:
IMHO, `preserve_empty_strings` option can't work because we remove unused fields from the form

Reply:
This makes sense IMO and we make we transform correctly where necessary in this MR.